hyper/hyper/restore_test.go

```diff
@@ -0,0 +1,57 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRestore(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) string
+		ExpectedErr    string
+	}{
+		"restore the desktop correctly": {
+			PrepareDesktop: func(h *hyper.Hyper) string {
+				desktop, err := h.Start(hyper.TestMinDesktopXML(t), &hyper.StartOptions{})
+				require.NoError(err)
+
+				err = h.Save(desktop, "test.dump")
```

Comment:
This should be saved in a temp directory that should be deleted after the tests (see `ioutil.TempDir("", "isard-test-")`

Reply:
ok